src/notification/notification-content.interface.ts

```diff
@@ -12,6 +12,7 @@ export interface NotificationContent {
 	showClose?: boolean;
 	lowContrast?: boolean;
 	template?: TemplateRef<any>;
+	templateData?: any;
```

Comment:
It might be more useful if we add an index type (https://www.typescriptlang.org/docs/handbook/advanced-types.html#index-types-and-index-signatures - `[key: any]: any`) and just passed the whole `NotificationContent` through to the template ... that way for simple cases like adjusting titles or captions we don't need to duplicate the values

Reply:
added a commit, is it what you were thinking of?